sdk-python/littlehorse/workflow.py

```diff
@@ -287,25 +403,49 @@ def __init__(self, workflow: "Workflow", initializer: "ThreadInitializer") -> No
         """
         self.wf_run_variables: list[WfRunVariable] = []
         self._workflow = workflow
-        self._nodes: dict[str, Node] = {}
+        self._nodes: list[WorkflowNode] = []
```

Comment:
Interesting, this is quite different from the other two SDK's. Probably it's cleaner this way.